### DISCUSSION TITLE
serial Treiber fehlen bei der 7490

### DISCUSSION THREAD

**Discussion by muckel-sassi** [Freetz-NG/freetz-ng#285] [Allgemein] [answered]
Hallo,
warum wurden die seriellen Treiber bei den letzten Updates entfernt?
Bis Changeset 18258 waren sie integriert und haben auch funktioniert

**Comment by fda77** [comment, marked as answer]
Es ist seit 4. Jan bekannt dass die Sourcen nicht mehr passen. Weshalb hast du dich bislang dicht darum gekümmert? Das sind 4 Monate!
-> https://github.com/Freetz-NG/freetz-ng/discussions/134#discussioncomment-259150
Es kann sein dass das 1 Modul das du benutzt funktioniert, bei anderen ist das nicht so

**Comment by muckel-sassi** [comment]
is ja gut, war ja nur ne Frage!!! Und ja, hat es....

**Comment by fda77** [comment]
Wenn du 100 mal die gleiche Frage beantworten musst hast du ab dem 10. mal auch keine Lust mehr. Deshalb ist die Diskussion auch angepinnt
